requirements.txt

```diff
@@ -0,0 +1,5 @@
+numpy
+# with mpi support
+h5py
+mpi4py
+gymnasium
```

Comment:
these files dont have their versions pinned, so its possible they have been updated since this MR was originally created & have caused breaking changes